demos/noise_suppression_demo/cpp/main.cpp

```diff
@@ -2,67 +2,46 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-/**
-* \brief The entry point for the Inference Engine noise_suppression_demo demo
-* \file noise_suppression_demo/main.cpp
-* \example noise_suppression_demo/main.cpp
-*/
-#include <climits>
-#include <vector>
-#include <string>
-#include <chrono>
-#include <iomanip>
-#include <iostream>
-#include <fstream>
-
-#include "openvino/openvino.hpp"
-
-#include "gflags/gflags.h"
-#include "utils/common.hpp"
-#include "utils/slog.hpp"
-
-typedef std::chrono::steady_clock Time;
-
-static const char help_message[] = "Print a usage message.";
-static const char inp_wav_message[] = "Required. Path to a input WAV file.";
-static const char out_wav_message[] = "Optional. Path to a output WAV file.";
-static const char model_message[] = "Required. Path to an .xml file with a trained model.";
-static const char target_device_message[] = "Optional. Specify the target device to infer on (the list of available "
-                                            "devices is shown below). Default value is CPU. "
-                                            "The demo will look for a suitable plugin for device specified.";
-DEFINE_bool(h, false, help_message);
-DEFINE_string(i, "", inp_wav_message);
-DEFINE_string(o, "noise_suppression_demo_out.wav", out_wav_message);
-DEFINE_string(m, "", model_message);
-DEFINE_string(d, "CPU", target_device_message);
-
-static void showUsage() {
-    std::cout << std::endl;
-    std::cout << "noise_suppression_demo [OPTION]" << std::endl;
-    std::cout << "Options:" << std::endl;
-    std::cout << std::endl;
-    std::cout << "    -h           " << help_message << std::endl;
-    std::cout << "    -i INPUT     " << inp_wav_message << std::endl;
-    std::cout << "    -o OUTPUT    " << out_wav_message << std::endl;
-    std::cout << "    -m MODEL     " << model_message << std::endl;
-    std::cout << "    -d DEVICE    " << target_device_message << std::endl;
-}
-
-bool ParseAndCheckCommandLine(int argc, char* argv[]) {
-    gflags::ParseCommandLineNonHelpFlags(&argc, &argv, true);
-    if (FLAGS_h) {
-        showUsage();
-        return false;
+#include <utils/common.hpp>
+#include <utils/slog.hpp>
+
+#include <gflags/gflags.h>
+#include <openvino/openvino.hpp>
```

Comment:
```suggestion
#include <openvino/openvino.hpp>

#include "gflags/gflags.h"
#include "utils/common.hpp"
#include "utils/slog.hpp"
```

Reply:
can we agree on some systematic headers listing?

Reply:
Reordered